static/docs/command-reference/run.md

```diff
@@ -54,23 +54,20 @@ try creating a pipeline.
 
 ### Well-behaved commands
 
-DVC is simple to use, you only need to wrap your commands with `dvc run`, and
-define your dependencies and outputs.
+In a sense, `dvc run` works as a special wrapper for your commands. To prevent
+unexpected behavior when DVC executes them, your commands should ideally follow
+these rules:
 
-However, to prevent unexpected behaviors, ideally, your commands should follow
-some principles:
+- Read exclusively from the specified dependencies.
+- Write exclusively to the specified outputs.
+- Completely rewrite outputs (i.e. do not append or edit).
+- Stop reading and writing files when the command exits.
 
-- Read exclusively from specified dependencies.
-- Write exclusively to specified outputs.
-- Completely rewrite the outputs (i.e. do not append or edit).
-- Stop reading and writing when the command exits.
-
-To guarantee reproducibilty, your command should be
+At the very least, to guarantee reproducibility, your command should be
 [deterministic](https://en.wikipedia.org/wiki/Deterministic_algorithm) (i.e. it
```

Comment:
I think this is the only part I don't have an objection to. But I would prefer to use less imperative language i.e. "**If** you want to guarantee exact reproducibility, ..." Thoughts?

Reply:
yep

Reply:
OK, changed.